conda_self/cli/main_migrate.py

```diff
@@ -3,10 +3,50 @@
 import sys
 from typing import TYPE_CHECKING
 
+from conda.cli.helpers import add_output_and_prompt_options
+from conda.reporters import confirm_yn
+
 if TYPE_CHECKING:
     import argparse
 
-HELP = "Protect 'base' environment from any further modifications"
+HELP = (
+    "Migrate your `base` environment into a new  environment named `default`"
+    " and protect your base."
```

Comment:
```suggestion
    "Protect `base` from accidental modifications and provide a modifiable copy "
    "that will be configured as default."
```

Better? Happy to hear alternatives. We can't assume the new one will be named `default`.